dashboard/src/pages/api/stripe.ts

```diff
@@ -0,0 +1,29 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import Stripe from 'stripe';
+
+import config from '@/server-config';
+
+const stripe = config.stripe ? new Stripe(config.stripe?.secretKey, { apiVersion: '2022-11-15' }) : undefined;
+
+async function isCustomer(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
discussed offline. guarding the `/run` endpoint should be enough to address the point here. I will do it in a followup PR to avoid developing against the build image CI check which takes over an hour

Reply:
@depombo since you had to modify the Dockerfile anyway, perhaps we just do this now?

Reply:
I thought the Docker build test wouldn't run anymore, but it does run because I am touching the `dashboard/yarn.lock`. I could have done it simultaneously when I undid the Dockerfile change, but I already didn't so perhaps still do it in a follow-up PR assuming the change works in staging?